security-malware-firewall.php

```diff
@@ -783,7 +783,7 @@ function spbc_deactivation($network)
                 $wpdb->prepare(
                     "DELETE FROM {$wpdb->postmeta}
 					WHERE meta_key = %s",
-                    'spbc_links_checked'
+                    '_spbc_links_checked'
```

Comment:
The old named meta (without underscore) need to be deleted too during deactivation.